ssl/ech/ech_helper.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/ssl.h>
+#include <openssl/ech.h>
+#include "../ssl_local.h"
+#include "ech_local.h"
+
+#ifndef OPENSSL_NO_ECH
+
+/* TODO(ECH): move code that's used by internals and test here */
+
+#endif
```

Comment:
Given you do not build ech subdir in build.info `#ifndef OPENSSL_NO_ECH` is not needed here and in the other files in ssl/ech

Reply:
ah, good point and will do

Reply:
will leave the`#ifndef` in `ech_local.h` as that'll later be included from e.g. `statem` source for things yet-to-be added to `ech_internal.c`